packages/web/src/app/doc-pages/components/components-doc.component.html

```diff
@@ -0,0 +1,85 @@
+<div class="e-title-md e-mb-40">Components</div>
+
+<elvia-radio-filter
+  [value]="filterValue"
+  [name]="'componentFilter'"
+  [items]="[
+    { label: 'All', value: 'all' },
+    { label: 'Navigation', value: 'navigation' },
+    { label: 'Action', value: 'action' },
+    { label: 'Input', value: 'input' },
+    { label: 'Data display', value: 'dataDisplay' },
+    { label: 'Feedback', value: 'feedback' },
+    { label: 'Layout', value: 'layout' }
+  ]"
+  (valueOnChange)="handleCategoryChange($any($event).target.value)"
+></elvia-radio-filter>
+
+<div class="e-mt-16">
+  @if (filterValue === 'all') {
+    <div class="components-container">
+      @for (component of allComponents; track component) {
+        <ng-container *ifViewportSize="['lg', 'xl']">
+          <a [routerLink]="getComponentPath(component)">
+            <elvia-card [heading]="getComponentTitle(component)" [maxHeadingLines]="2">
```

Comment:
Do note that this `getComponentTitle` method runs _very_ often and thereby runs its underlying `getDocPage` very often too. This might decrease performance 🧐. I'm not entirely sure how we can work around this at the moment... (It might not be a big deal 😄)

https://github.com/3lvia/designsystem/assets/54577949/8e2982ec-3cce-4096-939b-3bc540ef25ec

Reply:
Perhaps it would be better suited to be a pipe, rather than a getter 🤷